pkg/config/config.go

```diff
@@ -230,6 +230,10 @@ type PluginConfig struct {
 	// UnsetSeccompProfile is the profile containerd/cri will use If the provided seccomp profile is
 	// unset (`""`) for a container (default is `unconfined`)
 	UnsetSeccompProfile string `toml:"unset_seccomp_profile" json:"unsetSeccompProfile"`
+	// TolerateMissingHugePagesCgroupController if set to false will error out on create/update
+	// container requests with huge page limits if the cgroup controller for hugepages is not present.
+	// This helps with supporting Kubernetes <=1.18 out of the box. (default is `true`)
```

Comment:
should this be 1.19?

Reply:
@BenTheElder, we haven't released 1.19 yet right? So we could add similar code in 1.19 to detect and stop telling containerd/runc to do HugePages, so we can remove this flag in containerd at some point in time. Hence the `Kubernetes <=1.18` as we won't go back to fix older k8s already in the field.

Reply:
I could see changing the default to false at some point thus moving from the warning about skipping the limits to an error and at the same time we'd deprecate the flag. 

I think saying 1.18 is ok for the context of when it was pushed.

Reply:
Quick question: will containerd / CRI ignore unknown fields? Or will we have to make sure we only set this starting at some minimum version?

We'll attempt a kind upgrade as soon as this is vendored into the main repo.